Clients/src/presentation/components/ProjectCard/ProgressBar/index.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Progress bar display component for calculating the status of completed actions in fields.
+ *
+ * @component
+ * @param {ProgressBarProps} props - The properties for the ProgressBar component.
+ * @param {string} props.progress - The progress indicator as a fraction.
+ *
+ * @returns {JSX.Element} The rendered ProgressBar component.
+ */
+
+import { Box } from "@mui/material";
+import { FC } from "react";
+
+interface ProgressBarProps {
+    progress: string
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({progress}) => {
+    const progressCount = (progressString: string): number => {
+        const [completed, total] = progressString.split('/').map(Number);
+        if (isNaN(completed) || isNaN(total) || total === 0) {
+          console.error(`Invalid progress string: ${progressString}`);
```

Comment:
@popovaevgeniya Thank you so much you the this nice code 👍🏻🙏🏻
I guess we can make this a little bit better. 🤔
How about, as suggested with CodeRabbitAI, we replace `isNaN` with `Number.isNaN`, if possible.
And also, instead of logging the error, let throw the error. Like this:

`throw new Error("Invalid progress string: ${progressString}");`